### PR TITLE
fix(ecs): fix typo from fromServiceAtrributes to fromServiceAttributes

### PR DESCRIPTION
**Pull request by dafujii**
Fixed: #20458 

I found `fromServiceAtrributes`. I fixed to `fromServiceAttributes`

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/master/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by dafujii** [comment]
Ok, I'll try to fix the build.

File: packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts [hunk 1]
```diff
@@ -27,7 +27,15 @@ export interface ServiceAttributes {
...
 }
 
-export function fromServiceAtrributes(scope: Construct, id: string, attrs: ServiceAttributes): IBaseService {
+export function fromServiceAttributes(scope: Construct, id: string, attrs: ServiceAttributes): IBaseService {
+  return fromServiceAtrributes(scope, id, attrs);
+}
+
+/**
+ * DEPRECATED
+ * @deprecated use fromServiceAttributes()
```
Comment (peterwoodworth) [inline packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts:36]: I don't think we need to actually deprecate this. It's not exposed to the user, so this shouldn't be a breaking change we have to keep around. Can we only have the `fromServiceAttributes()` function?

File: deprecated_apis.txt [hunk 2]
```diff
@@ -748,6 +748,7 @@ constructs.Node#uniqueId
 @aws-cdk/aws-ecs.EcsOptimizedAmiProps#windowsVersion
 @aws-cdk/aws-ecs.FargateServiceProps#securityGroup
 @aws-cdk/aws-ecs.SplunkLogDriverProps#token
+@aws-cdk/aws-ecs.ServiceAttributes#fromServiceAtrributes
```
Comment (peterwoodworth) [inline deprecated_apis.txt:751]: Similarly to below, we won't be deprecating anything. No need to have this

Additionally, this file is for functionalities specifically deprecated from `v1` which no longer exist in `v2` - so we wouldn't want to add this anyway 🙂

**Review by peterwoodworth — CHANGES_REQUESTED** [review]
Thanks for the contribution @dafujii! couple things

File: packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts (hunk 1 above)
Comment (dafujii) [reply packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts]: Thanks, I did.

File: deprecated_apis.txt (hunk 2 above)
Comment (dafujii) [reply deprecated_apis.txt]: Thank you for your explanations😄

File: packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts [hunk 3]
```diff
@@ -114,7 +114,7 @@ export class FargateService extends BaseService implements IFargateService {
    * Imports from the specified service attrributes.
```
Comment (peterwoodworth) [inline packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts:114]: One more 🙂

Comment (dafujii) [reply packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts]: 😂 Thank you!!!!

**Review by peterwoodworth — APPROVED** [review]
lgtm!

**Pull request merged**